python/foglamp/common/logger.py

```diff
@@ -26,7 +26,8 @@
 
 
 def setup(logger_name: str = None,
-          destination: int = SYSLOG,
```

Comment:
Note that docker doesn't log to syslog naturally. 
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.5/site-packages/foglamp/services/core/__main__.py", line 10, in <module>
    from foglamp.services.core.server import Server
  File "/usr/local/lib/python3.5/site-packages/foglamp/services/core/server.py", line 19, in <module>
    from foglamp.services.core import routes as admin_routes
  File "/usr/local/lib/python3.5/site-packages/foglamp/services/core/routes.py", line 7, in <module>
    from foglamp.services.core.api import audit as api_audit
  File "/usr/local/lib/python3.5/site-packages/foglamp/services/core/api/audit.py", line 11, in <module>
    from foglamp.common.storage.payload_builder import PayloadBuilder
  File "/usr/local/lib/python3.5/site-packages/foglamp/common/storage/payload_builder.py", line 23, in <module>
    _LOGGER = logger.setup(__name__)
  File "/usr/local/lib/python3.5/site-packages/foglamp/common/logger.py", line 70, in setup
    handler = SysLogHandler(address='/dev/log')
  File "/usr/local/lib/python3.5/logging/handlers.py", line 806, in __init__
    self._connect_unixsocket(address)
  File "/usr/local/lib/python3.5/logging/handlers.py", line 834, in _connect_unixsocket
    self.socket.connect(address)
FileNotFoundError: [Errno 2] No such file or directory